custom_components/localtuya/pytuya/__init__.py

```diff
@@ -196,6 +196,8 @@ def __init__(self, dev_id, address, local_key, connection_timeout=10):
         self.dev_type = 'type_0a'
 
         self.port = 6668  # default - do not expect caller to pass in
+        
+        log.debug("Initiated TuyaDevice where self.id=%s self.address=%s self.local_key=%s self.connection_timeout=%s self.version=%s self.dev_type=%s self.port=%s", self.id, self.address, self.local_key, self.connection_timeout, self.version, self.dev_type, self.port)
```

Comment:
I would argue the this should be done by the user and not the library (I.e. this log point shouldn't be here).

Reply:
Yes, this was just a debug for me. I'll remove.

Reply:
Did you remove this?